platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -140,6 +142,13 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
+    @Override
+    public Collection<Describable> getOptionalMigratableInfo() {
+        return dataMigratables.stream()
```

Comment:
I'm all for using Lambda and everything, but I really do think this is going too far. Using `Collections.addAll()` or guava's `ImmutableList.of()` is, shorter, easier to understand and arguably faster than jumping through all those hoops just to copy a list.